radio/src/lua/api_mavlink.cpp

```diff
@@ -0,0 +1,224 @@
+/*
+ *  (c) www.olliw.eu, OlliW, OlliW42
```

Comment:
Please use the standard GPLv2 boiler plate. You can still place your copyright notice inside that GLP boilerplate if you want.